src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,24 +1,43 @@
-import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as filterActions } from '../../features/filter';
+
+import { Select } from '../../helpers/SelectEnum';
 
 export const TodoFilter: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const data = useAppSelector(state => state.filter);
+
+  const handleFunc = (e: React.ChangeEvent<HTMLInputElement>) => {
+    return dispatch(filterActions.filterTodo(e.target.value));
+  };
+
+  const handleSelectFunc = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    return dispatch(filterActions.selectTodo(e.target.value));
+  };
```

Comment:
1. Bad naming `handleFunc` and `handleSelectFunc`
2. These function should be responsible for setting `query` and `status` and it is not obvious from action creator name that they do this.